functions/validation/validation.js

```diff
@@ -140,6 +140,32 @@ function validateUniqueReleaseNames(releases) {
   return errors;
 }
 
+/**
+ * Validates that the release branch exists.
+ *
+ * @param {Object} release - The release to validate.
+ * @return {Array} errors - A list of errors from the validation of the release
+ * branch.
+ */
+function validateReleaseBranch(release) {
+  const errors = [];
+
+  if (!release.releaseBranchName) {
+    errors.push({
+      message: ERRORS.MISSING_RELEASE_FIELD,
+      offendingRelease: release,
+    });
+  } else if (typeof release.releaseBranchName !== "string" ||
+          release.releaseOperator.trim() === "") {
+    errors.push({
+      message: ERRORS.INVALID_RELEASE_FIELD,
+      offendingRelease: release,
+    });
```

Comment:
On today's episode of _"fighting the linter"_:
```js
const isNotString = typeof release.releaseBranchName !== "string";
const isEmpty = release.releaseOperator.trim() === ""

if (!release.releaseBranchName) {
  errors.push({
    message: ERRORS.MISSING_RELEASE_FIELD,
    offendingRelease: release,
  });
} else if (isNotString || isEmpty) {
  errors.push({
    message: ERRORS.INVALID_RELEASE_FIELD,
    offendingRelease: release,
  });
}
```

although, it's a bit of a design choice and not a big deal. feel free to keep it as is if you feel it reads easier.